weed/command/server.go

```diff
@@ -147,6 +147,8 @@ func init() {
 	s3Options.auditLogConfig = cmdServer.Flag.String("s3.auditLogConfig", "", "path to the audit log config file")
 	s3Options.allowEmptyFolder = cmdServer.Flag.Bool("s3.allowEmptyFolder", true, "allow empty folders")
 	s3Options.allowDeleteBucketNotEmpty = cmdServer.Flag.Bool("s3.allowDeleteBucketNotEmpty", true, "allow recursive deleting all entries along with bucket")
+	s3Options.filerGroup = cmdServer.Flag.String("filerGroup", "", "filerGroup of filer to use as prefix if collectionPrefix is enable")
+	s3Options.collectionPrefix = cmdServer.Flag.Bool("collectionPrefix", false, "use filerGroup as collection prefix for buckets")
```

Comment:
I prefer to avoid this option and always use filer group as prefix.

Reply:
What about when you want to have a filer with filerGroup and s3 enabled?  
for example:  
```
weed filer  -filerGroup abc -s3
```
If we use filerGroup as a prefix in this example our change is not backward compatible  
maybe use `s3.collectionPrefix` but don't use this flag in `weed s3`?

Reply:
filerGroup is rarely used I would say. We can do this non-backward compatible change. If customers really want to use this, they can rename existing collections by prefixing the filerGroup to the collection name.